extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1427,9 +1439,13 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     // unlayoutCallback execution.  Assume that if called for one slot, it will
     // be called for all and we should cancel SRA execution.
     const checkStillCurrent = this.verifyStillCurrent();
-    const noFallbackExp = this.experimentIds.includes(
-      DOUBLECLICK_SRA_EXP_BRANCHES.SRA_NO_RECOVER
-    );
+    const noFallbackExp =
+      this.experimentIds.includes(
+        DOUBLECLICK_SRA_EXP_BRANCHES.SRA_NO_RECOVER
+      ) ||
+      this.experimentIds.includes(
+        RANDOM_SUBDOMAIN_BRANCHES.RANDOM_SUBDOMAIN_EXPERIMENT
```

Comment:
@keithwrightbos Something like this?

Reply:
See previous comment.  This is unnecessary here... instead place within getSafeframe override

Reply:
removed